modules/special-rules/partials/summary-of-super-heavy-weapon-effects.adoc

```diff
@@ -0,0 +1,5 @@
+.Summary of super-heavy weapon effects
+** When your detachment's Firepower is halved for any reason, roll to *lock on* with each of its super-heavy weapons.
+** Super-heavy weapons may contribute to the placement of Blast markers in one of two main ways, depending on whether or not they contribute Firepower in the attack.
+** After Firepower, each remaining Blast marker suppresses one super-heavy weapon.
+** In a firefight, a super-heavy weapon that lacks a basic Firepower value contributes 1 Firepower.
```

Comment:
What super-heavy weapons have a basic Firepower value? Is Anti-personnel the only one? And should we treat even this in the same way (especially when you consider that the corresponding ability states "For the purposes of suppression by Blast markers and such, treat the unit as having a Firepower value of 1.")

Reply:
I think it's just Anti-Personnel, but that one would be better handled as 'Anti-Personnel (X)' anyway, as the unit doesn't actually have a normal firepower value any more...

Reply:
Strictly, they still have a Firepower value, but we apply the AP rules rather than use the Firepower value in the normal way. But I suppose you're saying that another approach would achieve the same end result? But isn't it tricky because it is both an ability and a super-heavy weapon?
Would you make the ability the '(X)' part, or the super-heavy weapon itself? I'm guessing the ability should be the '(X)' part, and then if a war engine has the SHW we'd show the number of shots on the data sheet as normal, rather than use the ability with the '(X)' part?

It doesn't have to hold us up here of course, but I'm just thinking about it because of the non-sequitur in "super-heavy weapon that lacks a basic Firepower value".

Reply:
Bear in mind that the ability part is just a vehicle variant ability saying 'it has an Anti-Personnel weapon' which is used by one vehicle that GW hand-waved as costing the same as a normal Leman Russ. We can remove the AP ability and it would have absolutely zero impact on the game if we list the unit's data sheet directly.

I know it looks like that comment goes against the stuff I've said previously about data sheet modifiers, but Vehicles are different because there are comparatively few of them.

> Would you make the ability the '(X)' part, or the super-heavy weapon itself?

Personally, I'd remove the ability completely, and the weapon would be Anti-Personnel (X).

> Strictly, they still have a Firepower value, but we apply the AP rules rather than use the Firepower value in the normal way.

> It doesn't have to hold us up here of course, but I'm just thinking about it because of the non-sequitur in "super-heavy weapon that lacks a basic Firepower value".

These are two of the reasons why I _loath_ the Anti-Personnel SHW. We get told to 'ignore their original Firepower value and **instead** count as being armed with an Anti-personnel super heavy weapon' (my emphasis) which means that they no longer have a numerical Firepower value, and then get told to 'roll a number of dice equal to the units Firepower' which they no longer have.

Aaaaaaaargh.

Reply:
>I know it looks like that comment goes against the stuff I've said previously about data sheet modifiers, but Vehicles are different because there are comparatively few of them.

I think I see what you're saying. So, we'd do away with the ability and list any AP-bearing units directly, with 'Anti-personnel (X)', similar to how a Land Raider is an independent entry with '2 x Macro-weapon'? Yes, that seems like the lesser of the evils to me.

>These are two of the reasons why I loath the Anti-Personnel SHW. We get told to 'ignore their original Firepower value and instead count as being armed with an Anti-personnel super heavy weapon' (my emphasis) which means that they no longer have a numerical Firepower value, and then get told to 'roll a number of dice equal to the units Firepower' which they no longer have.

Well, we see what they mean I think — essentially, 'ignore the value for **most** purposes, but keep it in mind as the reference value for this one purpose that relates to the ability'. But yes, they didn't write it precisely and it grates -- and I suspect that a precise version would still feel quite clumsy. I also loathe having two different rules with the same name (hence [weapon] and [ability] in the titles, to disambiguate, but I still loathe it).

And after all that, the ability doesn't even work very well (if my hazy recollection of some Facebook conversations is correct). But I'm sure we could find a way to make it worthwhile -- did those conversations ever reach a conclusion that seemed worthy?

Anyway, back to the point, this should work shouldn't it? (With corresponding tweak in the main topic of course.)
```suggestion
** In a firefight, a super-heavy weapon contributes 1 Firepower.
```
And then, do you think we might as well remove the AP ability (and give the AP SHW minimum necessary tweaks to match) in this branch? Or leave these for later and also improve the actual AP SHW rules at the same time under another issue and branch?

Reply:
>And then, do you think we might as well remove the AP ability (and give the AP SHW minimum necessary tweaks to match) in this branch? Or leave these for later and also improve the actual AP SHW rules at the same time under another issue and branch?

Actually, I've just remembered that Anti-personnel was an 'experimental' rule anyway. So, as a minimum I think we could move it to the experimental section and not worry about it as far as the sentence I suggested goes. Or as I say, just delete it and improve the rest in a later effort.

Reply:
The suggestion sounds good anyway!

But I agree, move the ability and weapon to the experimental section for now, and we can revisit it at a later date.

Reply:
>The suggestion sounds good anyway!

Sorry, I mean 'apply the suggestion but not worry about whether or not AP present a slight edge case to contradict it'. :) Okay, I'll apply the suggestion and move the AP bits to the Experimental section for now.